src/sentry/static/sentry/app/components/acl/access.tsx

```diff
@@ -2,69 +2,92 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import {t} from 'app/locale';
+import {Config, Organization, User} from 'app/types';
 import Alert from 'app/components/alert';
 import SentryTypes from 'app/sentryTypes';
 import withConfig from 'app/utils/withConfig';
+import withOrganization from 'app/utils/withOrganization';
 
 const DEFAULT_NO_ACCESS_MESSAGE = (
   <Alert type="error" icon="icon-circle-info">
     {t('You do not have sufficient permissions to access this.')}
   </Alert>
 );
 
+// Props that function children will get.
+export type ChildRenderProps = {
+  hasAccess: boolean;
+  hasSuperuser: boolean;
+};
+
+type ChildFunction = (props: ChildRenderProps) => null | React.ReactNode;
```

Comment:
Is `null` not a `ReactNode`?

Reply:
I'll double check as it should be. I had some trouble getting this all to compile though.